pages/spaces/dialogDemo.js

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import Button from "@material-ui/core/Button";
+import Avatar from "@material-ui/core/Avatar";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemAvatar from "@material-ui/core/ListItemAvatar";
+import ListItemText from "@material-ui/core/ListItemText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Dialog from "@material-ui/core/Dialog";
+import PersonIcon from "@material-ui/icons/Person";
+import AddIcon from "@material-ui/icons/Add";
+import Typography from "@material-ui/core/Typography";
+
+import {
+  useTheme,
+  createMuiTheme,
+  ThemeProvider,
+} from "@material-ui/core/styles";
+import { Container, Grid, Box } from "@material-ui/core";
+import SpacesCard from "../../components/SpacesCard";
```

Comment:
how're you arranging your imports? I usually add all built-in/important libraries at the top and anything local separate on another block